### PR TITLE
[Storage] Bump azure-mgmt-storage version to use api version 2019-06-01

### PR DESCRIPTION
**Pull request by Juliehzl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/recordings/test_hdinsight_cluster_kafka_with_rest_proxy.yaml [hunk 1]
```diff
@@ -244,7 +244,7 @@ interactions:
       accept-language:
       - en-US
     method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/hdicli-000001/providers/Microsoft.Storage/storageAccounts/hdicli000002?api-version=2019-04-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/hdicli-000001/providers/Microsoft.Storage/storageAccounts/hdicli000002?api-version=2019-06-01
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/recordings/test_hdinsight_cluster_kafka_with_rest_proxy.yaml]: Did you modify the recording file manually?

Comment (Juliehzl) [reply src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/recordings/test_hdinsight_cluster_kafka_with_rest_proxy.yaml]: yes. It has some region requirements for cluster and I cannot create a valid cluster in live mode. in this way I have to replace storage related version to make ci pass.

File: src/azure-cli/setup.py [hunk 2]
```diff
@@ -122,7 +122,7 @@
     'azure-mgmt-signalr~=0.3.0',
     'azure-mgmt-sql~=0.15.0',
     'azure-mgmt-sqlvirtualmachine~=0.5.0',
-    'azure-mgmt-storage~=5.0.0',
+    'azure-mgmt-storage~=7.0.0',
```
Comment (qwordy) [inline src/azure-cli/setup.py]: Skip 6.0.0?

Comment (Juliehzl) [reply src/azure-cli/setup.py]: nothing. 7.0.0 is the latest one.

**Pull request merged**